### PR TITLE
Test22

### PR DESCRIPTION
**Pull request by yarikbratashchuk**

### CODE REVIEW

File: src/components/CompanyListItem/CompanyListItem.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+const CompanyListItem = ({ item, showEmployee, employee, toggleEmployeeFunc }) => (
+  <li className="list-group-item">
+    <span style={{ marginLeft: '10px', marginRight: '10px' }}>
+      <span>
+        {item.get('companyName')}
+      </span>
+      <button onClick={() => toggleEmployeeFunc()} >{ showEmployee ? 'Hide employee' : 'Show employee' }</button>
+      <ul>
+        {
+          showEmployee ? employee.map(emp => <Employee key={emp.get('id')} item={emp} companyId={item.get('id')} />) : ''
```
Comment (Architektor) [inline src/components/CompanyListItem/CompanyListItem.js]: use linter plzzz

Comment (yarikbratashchuk) [inline src/components/CompanyListItem/CompanyListItem.js]: I use linter. It says nothing in this file.

Comment (Architektor) [inline src/components/CompanyListItem/CompanyListItem.js]: http://eslint.org/docs/rules/max-len

Comment (yarikbratashchuk) [inline src/components/CompanyListItem/CompanyListItem.js]: https://github.com/web-pal/react-redux-boilerplate/blob/master/.eslintrc
there is no such rule in boilerplate.

Comment (Architektor) [inline src/components/CompanyListItem/CompanyListItem.js]: I'm sure it IS
https://github.com/web-pal/react-redux-boilerplate/blob/master/.eslintrc#L3
https://github.com/airbnb/javascript#whitespace--max-len

Comment (VladimirPal) [inline src/components/CompanyListItem/CompanyListItem.js]: Да вы заебали! Я блять сам разберусь по линтеру тут или нет, работаем нах.

**Pull request closed without merging**